Library/ViewModels/PledgeAmountViewModel.swift

```diff
@@ -185,11 +187,7 @@ public final class PledgeAmountViewModel: PledgeAmountViewModelType,
       )
     }
 
-    self.stepperValue = Signal.merge(
-      minValue,
-      self.amount.map { $0.0 }
-    )
-    .skipRepeats()
+    self.stepperValue = self.notifyDelegateAmountUpdated.map { $0.0 }.skipRepeats()
```

Comment:
There's no real need to `merge` with `minValue` - this is already taken into account in `notifyDelegateAmountUpdated`.